gravitee-apim-portal-webui/src/app/pages/application/application-general/application-general.component.ts

```diff
@@ -138,13 +143,19 @@ export class ApplicationGeneralComponent implements OnInit, OnDestroy {
             redirect_uris: new FormArray([]),
             grant_types: new FormArray([]),
           }),
+          tls: this.formBuilder.group({
+            client_certificate: new FormControl(this.application.settings.tls.client_certificate, null),
```

Comment:
`this.application.settings.tls` is never undefined ? 
because `tls?: TlsClientSettings;` say it's possible 

maybe we can use lodash `get(this.application, 'settings.tls.client_certificate')` ?